### PR TITLE
feat: add inv function

### PR DESCRIPTION
**Pull request by andybeltz**
### Summary

Added `Update-AriaNetworksNsxtDataSourceCredentials` cmdlet to update the credentials for a NSX Manager in VMware Aria Operations for Networks.

### Type

- [ ] Bugfix
- [X] Enhancement or Feature
- [ ] Code Style or Formatting
- [ ] Documentation
- [ ] Refactoring
- [ ] Chore
- [ ] Other
        Please describe:

### Breaking Changes?

- [ ] Yes, there are breaking changes.
- [X] No, there are no breaking changes.

### Test and Documentation

- [X] Tests have been completed.
- [X] Documentation has been added or updated.

### CODE REVIEW

File: docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksNsxtDataSourceCredentials.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+The `Update-AriaNetworksNsxtDataSourceCredentials` cmdlet allows a user to update credentials for a NSX Manager data source in VMware Aria Operations for Networks.
+
+## Examples
+
+### Example 1
+
+```powershell
+Update-AriaNetworksNsxtDataSourceCredentials -fqdn sfo-m01-nsx01.sfo.rainpole.io -id 15832:904:7312957441829059413 -CollectorId 15832:901:1711011916294613031 -nickname "sfo-m01-vc01 - Management Domain vCenter Server" -certificate F:\certs\sfo-m01-nsx01.cer -privatekey F:\certs\sfo-m01-nsx01.key
+This example updates the credentials for the NSX Manager data source in VMware Aria Operations for Networks to use a new certificate and private key for a NSX principal identity user.
```
Comment (tenthirtyam) [inline docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksNsxtDataSourceCredentials.md:33]: Move the description of the example outside of the code block. 🙃

File: docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksNsxtDataSourceCredentials.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+```
+
+### Example 2
+
+```powershell
+Update-AriaNetworksNsxtDataSourceCredentials -fqdn sfo-m01-nsx01.sfo.rainpole.io -id 15832:904:7312957441829059413 -CollectorId 15832:901:1711011916294613031 -nickname "sfo-m01-vc01 - Management Domain vCenter Server" -username svc-inv-vsphere -password VMw@re1!VMw@re1!
+This example updates the credentials for the NSX Manager data source in VMware Aria Operations for Networks to use a new NSX service account user with a password.
```
Comment (tenthirtyam) [inline docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksNsxtDataSourceCredentials.md:40]: Move the description of the example outside of the code block. 🙃

**Review by tenthirtyam — CHANGES_REQUESTED** [review]
Just need to move the descriptions in the examples outside of the code blocks. 🙃

File: docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksNsxtDataSourceCredentials.md (hunk 2 above)
Comment (andybeltz) [reply docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksNsxtDataSourceCredentials.md]: Fixed in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/cd3f7c173de4e0535b7ea866786ba1f78d1ae24e

**Review by tenthirtyam — APPROVED** [review]
LGTM!

**Pull request merged**